package.json

```diff
@@ -46,7 +46,7 @@
   "peerDependencies": {
     "@folio/stripes": "^3.0.0",
```

Comment:
You want to upgrade `stripes` to `v4` in your peer-deps and dev-deps too: 
```
"@folio/stripes": "^4.0.0",
```

Reply:
Sure, created a PR for these changes:
https://github.com/folio-org/ui-notes/pull/69